dist/tags/features/offerwall-dynamic-ui.html

```diff
@@ -28,15 +29,18 @@
   };
   // #endregion
 
-  if (!('MutationObserver' in window)) {
-    return;
-  }
-
   var FEATURE_ID = 'offerwallDynamicUi';
   var variant = '{{tagsmith.abVariant.offerwallDynamicUi}}';
   var enabled = window.__tagsmith.enable(FEATURE_ID, variant);
   var logger = window.__tagsmith.getLogger(FEATURE_ID, variant);
 
+  var $root = document.getElementById('__tagsmith_offerwallDynamicUi');
+
+  if ((!enabled && !logger) || !('MutationObserver' in window)) {
```

Comment:
[IMO] Shouldn't we add a test for removing the root when 'MutationObserver is not available?
https://github.com/google-marketing-solutions/tagsmith/pull/8/files?diff=split&w=0#diff-7b75a77b7b9a58746ec34ba843e0589e338d9ae12f6e9663230b1d8e063d57ffR41

Reply:
Added test